SCSS/_global.scss

```diff
@@ -0,0 +1,44 @@
+body {
+    font-family: 'Raleway', sans-serif;
+}
+
+.section {
+    
+    &--headline {
+        text-align: center;
+        margin-top: 40px;
+        margin-bottom: 20px;
+
+        &__title {
+            font-size: 34px;
+            text-transform: capitalize;
+        }
+
+        &__subtitle {
+            font-size: 18px;
+            color: $subtitle-color;
+
+            &:before {
```

Comment:
a może zamiast powielać te same style dla after oraz before użyć @extends? Przykładowy opis: https://www.sitepoint.com/the-benefits-of-inheritance-via-extend-in-sass/

Reply:
zrobione, fajna sprawa :)